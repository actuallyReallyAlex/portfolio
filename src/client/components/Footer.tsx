import * as React from "react";

export interface FooterProps {}

const Footer: React.FunctionComponent<FooterProps> = () => {
  return <footer>Alex Lee | Alex Lee Â© 2020</footer>;
};

export default Footer;
