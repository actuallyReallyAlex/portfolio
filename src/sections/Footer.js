import React from 'react'
import { Box, Heading, Text } from 'grommet'
import SocialIcon from '../components/SocialIcon'
import { social } from '../config.json'

const Footer = () => {
  return (
    <Box
      align="center"
      className="primary-color"
      direction="row"
      fill="horizontal"
      height="50px"
      justify="between"
      pad={{ horizontal: 'medium' }}
    >
      <Text color="white" size="xlarge" weight="bold">
        Alex Lee
      </Text>
      <Box align="center" direction="row" gap="medium">
        <Text color="white">Â© 2019 Alex Lee</Text>
        <Box direction="row" gap="small">
          {social.map(({ icon, link }, i) => (
            <SocialIcon key={i} icon={icon} link={link} size="medium" />
          ))}
        </Box>
      </Box>
    </Box>
  )
}

export default Footer
