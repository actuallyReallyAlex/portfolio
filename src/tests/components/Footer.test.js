import React from 'react'
import Footer from '../../components/Footer'
import { shallow } from 'enzyme'

const defaultProps = {}

describe('<Footer />', () => {
  afterEach(() => {
    // defaultProps.handler.mockClear()
  })

  test('Should render the <Footer /> component.', () => {
    const component = shallow(<Footer {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
