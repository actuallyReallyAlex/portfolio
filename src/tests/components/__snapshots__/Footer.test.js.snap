// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should render the <Footer /> component. 1`] = `ShallowWrapper {}`;
